db/migrate/20170508150135_add_foreign_keys.rb

```diff
@@ -0,0 +1,31 @@
+class AddForeignKeys < ActiveRecord::Migration
+  def change
+    add_index :notifications, [:subject_id, :subject_type]
+    add_index :order_details, :project_id
+    add_index :order_statuses, :parent_id
+    add_index :orders, :merge_with_order_id
+    add_index :price_group_members, :account_id
+    add_index :products, :initial_order_status_id
+
+    add_foreign_key :account_users, :users
+    add_foreign_key :journal_rows, :accounts
+    add_foreign_key :journal_rows, :journals
+    add_foreign_key :journal_rows, :order_details
```

Comment:
I _think_ mysql complains that you can't specify journal rows as the update table and the from table, don't know about Oracle. Might have to do it in a two-line script.